core/src/test/java/org/kohsuke/stapler/ClassDescriptorTest.java

```diff
@@ -48,8 +39,8 @@
         for (Map.Entry<String,String[]> entry : expected.entrySet()) {
             Method testMethod = testCases.get(entry.getKey());
             assertNotNull("Method missing for " + entry.getKey(), testMethod);
-            String[] result = (String[])lpfa.invoke(null, testMethod);
-            assertNotNull("Null result for " + entry.getKey());
+            String[] result = ClassDescriptor.loadParameterNamesFromReflection(testMethod);
+            assertNotNull("Null result for " + entry.getKey(), result);
```

Comment:
Note that this fixes a test bug: this line was previously doing nothing, and a null `result` just led to an opaque NPE on the next line.